### ISSUE TITLE
Grad of model parameters is None

### ISSUE THREAD

**Issue by wubowen416** [facebookresearch/higher#69]
Hi, I found this repo to be very useful, but there is something problem with my usage. And since I could not figure out, so could you please help me on this?

Currently I'm try to test the grad of model parameters, so I use 1 dim input and output, and also 1 dim weight. 

The computation is just y = w*x. Here is my code:

```python
import torch
import torch.nn as nn
import torch.optim as optim
import torch.nn.functional as F

import higher as H

# configure in and out and model

x = torch.ones(1, 1)
y = torch.ones(1, 1)

model = nn.Linear(1, 1, bias=False)
for p in model.parameters():
    nn.init.constant_(p, 0.5)

criterion = nn.MSELoss()
opt = optim.SGD(model.parameters(), lr=1e-1)

with H.innerloop_ctx(model, opt) as (fmodel, diffopt):
    
    # unrolling loop
    out = fmodel(x)
    loss = criterion(out, y)
    diffopt.step(loss)

    # update params
    out = fmodel(x)
    error = criterion(out, y)
    error.backward()

optimizer.step()

print('Model params:')
for p in model.parameters():
    print(p)

print('fModel params:')
for p in fmodel.parameters():
    print(p)
    
print('grad of model weight:')
print(model.weight.grad)
```

here is the output:

```python
Parameter containing:
Model params:
Parameter containing:
tensor([[0.5000]], requires_grad=True)
fModel params:
tensor([[0.6000]], grad_fn=<AddBackward0>)
grad of model weight:
None
```

Thank you very much in advance.

**Comment by egrefen** [comment]
If you are trying to do something like MAML, i.e. backprop to the parameters of the model at the beginning of the unrolled loop, you need to instantiate the context manager with kwarg `copy_initial_weights=False` (see docs at https://higher.readthedocs.io/en/latest/toplevel.html).

I realise this is a bit confusing, and we're going to rename some elements of the API to make usage clearer in an upcoming version.

**Comment by wubowen416** [comment]
It's solved. Thank you!
